### PR TITLE
Updated Turkish translation file

### PR DESCRIPTION
**Pull request by yakuter**
Added the missing words in Turkish language file according to English language file.

### CODE REVIEW

File: publishable/lang/tr/voyager.php [hunk 1]
```diff
@@ -267,31 +281,61 @@
...
-        'table_no_columns'      => 'Tabloda hiç kolan bulunamadı...',
-        'type'                  => 'Tip',
-        'type_not_supported'    => 'Bu tip desteklenöiyor',
-        'unique'                => 'BENZERSİZ',
-        'unknown_type'          => 'Bilinmeyen Tip',
-        'update_table'          => 'Tabloyu güncelle',
-        'url_slug'              => 'Link yazısı (benzersiz olmalı)',
-        'url_slug_ph'           => 'Link yazısı (ex. gonderi)',
-        'visibility'            => 'Görünür',
+        'no_composites_warning'     => 'This table has composite indexes. Please note that they are not supported '.'at the moment. Be careful when trying to add/remove indexes.',
```
Comment (fletch3555) [inline publishable/lang/tr/voyager.php]: This line is still in English. Did you intend to translate it?

Comment (yakuter) [reply publishable/lang/tr/voyager.php]: Oh sorry, I really didn't notice it. Translating it now.

**Comment by yakuter** [comment]
Fixed the commented issue.

**Review by mhanoglu — APPROVED** [review]
Nice job. Thanks.

**Review by ysb — APPROVED** [review]

**Review by emir — APPROVED** [review]
👍

File: publishable/lang/tr/voyager.php [hunk 2]
```diff
@@ -267,31 +281,61 @@
...
-        'success_remove_bread'  => 'Başarılı silindi BREAD şurdan :datatype',
-        'success_update_bread'  => 'Başarılı güncellendi :datatype BREAD',
-        'success_update_table'  => 'Başarılı tablo güncellendi :table table',
-        'table_actions'         => 'Tablo işlemleri',
-        'table_columns'         => 'Tablo kolanları',
-        'table_has_index'       => 'Tablo zaten birincil anahtarı var.',
-        'table_name'            => 'Tablo Adı',
-        'table_no_columns'      => 'Tabloda hiç kolan bulunamadı...',
-        'type'                  => 'Tip',
-        'type_not_supported'    => 'Bu tip desteklenöiyor',
```
Comment (aozen) [inline publishable/lang/tr/voyager.php:288]: letter mistake.
'Bu tip desteklenmiyor'

File: publishable/lang/tr/voyager.php [hunk 3]
```diff
@@ -267,31 +281,61 @@
...
-        'unique'                => 'BENZERSİZ',
-        'unknown_type'          => 'Bilinmeyen Tip',
-        'update_table'          => 'Tabloyu güncelle',
-        'url_slug'              => 'Link yazısı (benzersiz olmalı)',
-        'url_slug_ph'           => 'Link yazısı (ex. gonderi)',
```
Comment (aozen) [inline publishable/lang/tr/voyager.php:293]: letter mistake
gönderi

**Review by aozen — CHANGES_REQUESTED** [review]

**Comment by fletch3555** [comment]
We're no longer accepting changes for the 1.0 branch since we're so close to a 1.1 release.  Can you please update the PR to resolve the conflicts? Language files have been broken out into separate files by top-level key, instead of being in one large file.

**Comment by marktopper** [comment]
Closed since conflicts not resolved.

**Pull request closed without merging**